app/services/visit.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+from fastapi import HTTPException, status
+from app.schemas import Doctor, VisitInfoHistory, VisitHistory
+from app.models import Client as ClientDB
+from app.logger import log
```

Comment:
from typing import List
from fastapi import HTTPException, status

from app.schemas import Doctor, VisitInfoHistory, VisitHistory
from app.models import Client as ClientDB
from app.logger import log

Reply:
Thank you) Appropriate remark) I will correct now)